### DISCUSSION TITLE
request - LURO My rounds -

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#63] [Bugs and Reports]
can i open my round just by click on the line ? Same like on PREDICT. not just on the small cross icon on the end of the line. 
<img width="1015" alt="Snímek obrazovky 2024-10-09 v 23 17 05" src="https://github.com/user-attachments/assets/8ffb0089-64ab-46ac-b4a6-46bbd83a507d">

**Comment by bf-monster** [comment]
Shoud be fixed now!
